src/components/top-bar/top-bar.atoms.tsx

```diff
@@ -0,0 +1,15 @@
+import type { FC, HTMLAttributes } from 'react'
+import { ElTopBarMainNav } from './styles'
+
+export const TopBarMainNav: FC<HTMLAttributes<HTMLDivElement>> = ({ children, ...props }) => {
+  return (
+    <ElTopBarMainNav
+      style={{
+        containerType: 'inline-size',
```

Comment:
**question:** is there any reason this is not part of `ElTopBarMainNav`'s styles?

Reply:
yes, there is a typing issue currently

![image](https://github.com/user-attachments/assets/3287d65c-51b3-4cc8-95f0-0bff1eecca66)

Reply:
**todo:** Can you add a comment explaining that then 🙏

Reply:
yes, no problem

Reply:
After I investigate further apparently its just IDE specific issue as I've tried another editor it doesn't show that warning, I've run yarn check its seems ok, I think I will move the style to the `ElTopBarMainNav` as you suggest